modules/outbound-command-event-handler/src/domain/bulk_batch_entity.ts

```diff
@@ -0,0 +1,208 @@
+    }
+    get bulkQuoteId(): string {
+        return this._state.bulkQuoteId;
+    }
+    get bulkTransferId(): string {
+        return this._state.bulkTransferId;
+    }
+
+    get bulkQuotesRequest(): SDKSchemeAdapter.Outbound.V2_0_0.Types.bulkQuoteRequest {
+        return this._state.bulkQuotesRequest;
+    }
+    get bulkQuotesResponse(): SDKSchemeAdapter.Outbound.V2_0_0.Types.bulkQuoteResponse | undefined {
+        return this._state.bulkQuotesResponse;
+    }
+    get bulkTransfersRequest(): SDKSchemeAdapter.Outbound.V2_0_0.Types.bulkTransferRequest {
+        return this._state.bulkTransfersRequest;
+    }
+    get bulkTransfersResponse(): SDKSchemeAdapter.Outbound.V2_0_0.Types.bulkQuoteResponse | undefined {
+        return this._state.bulkTransfersResponse;
+    }
+
+    private static _convertPartyToFrom(party: SDKSchemeAdapter.Outbound.V2_0_0.Types.Party): SDKSchemeAdapter.Outbound.V2_0_0.Types.bulkQuoteRequest['from'] {
+        return {
+            idType: party.partyIdInfo.partyIdType,
+            idValue: party.partyIdInfo.partyIdentifier,
+            idSubValue: party.partyIdInfo.partySubIdOrType,
+            displayName: party.name,
+            firstName: party.personalInfo?.complexName?.firstName,
+            middleName: party.personalInfo?.complexName?.middleName,
+            lastName: party.personalInfo?.complexName?.lastName,
+            dateOfBirth: party.personalInfo?.dateOfBirth,
+            merchantClassificationCode: party.merchantClassificationCode,
+            fspId: party.partyIdInfo.fspId,
+            extensionList: party.partyIdInfo.extensionList?.extension
+        }
+    }
+
+    static CreateEmptyBatch(
+        bulkTransactionEntity: BulkTransactionEntity
+    ): BulkBatchEntity {
+
+        const bulkQuoteId = randomUUID();
+        const bulkTransferId = randomUUID();
+
+        const initialState: BulkBatchState = {
+            id: randomUUID(),
+            state: BulkBatchInternalState.CREATED,
+            bulkId: bulkTransactionEntity.id,
+            bulkQuoteId,
+            bulkTransferId,
+            bulkQuotesRequest: {
+                homeTransactionId: bulkTransactionEntity.bulkHomeTransactionID,
+                bulkQuoteId,
+                from: BulkBatchEntity._convertPartyToFrom(bulkTransactionEntity.from),
+                individualQuotes: [],
+                extensions: bulkTransactionEntity.extensions
+            },
+            bulkTransfersRequest: {
+                homeTransactionId: bulkTransactionEntity.bulkHomeTransactionID,
+                bulkTransferId,
+                from: BulkBatchEntity._convertPartyToFrom(bulkTransactionEntity.from),
+                individualTransfers: [],
+                extensions: bulkTransactionEntity.extensions
+            },
+            quoteIdReferenceIdMap: {},
+            transferIdReferenceIdMap: {},
+            created_at: Date.now(),
+            updated_at: Date.now(),
+            version: 1,
```

Comment:
Define the `version` as a static class member, and use that to assign it here.

Reply:
Done